src/features/navigation/RootNavigator/Header/tests/Header.web.test.tsx

```diff
@@ -55,15 +67,8 @@ describe('Header', () => {
   })
 })
 
-function renderHeader({
-  isLoggedIn,
-  isBeneficiary,
-}: {
-  isLoggedIn: boolean
-  isBeneficiary: boolean
-}) {
-  mockedUseAuthContext.mockReturnValue({ isLoggedIn })
-  mockedUseUserProfileInfo.mockReturnValue({ data: { isBeneficiary } })
+function renderHeader({ isLoggedIn }: { isLoggedIn: boolean }) {
+  mockedUseAuthContext.mockReturnValue({ isLoggedIn, user: { isBeneficiary: true } })
```

Comment:
Hmmmm… On a un mockReturnValue dans le render, qui est appelé après un mockReturnValueOnce dans les it()… 
De 1, c'est pas ouf car on appelle pas que Once dans nos it(), et de 2, ça annule les Once de nos it() non ? 🤔